### PR TITLE
added LPWC website

### PR DESCRIPTION
**Pull request by thevaachandereng**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ It is designed to identify groups of biological entities (for example, genes or
 LPWC allows lags to incorporate delayed responses in the biological data.
 For example, two genes may have similar expression changes over time, but one initiates those changes 5 minutes after the other.
 LPWC also supports irregular time intervals between time points collected in biological data.
+LPWC website is available [here](https://thevaachandereng.github.io/LagPenalizedWeightedCorrelation/).
```
Comment (agitter) [inline README.md]: Can you change `https://thevaachandereng.github.io/LagPenalizedWeightedCorrelation/` to the new URL that will go live once we merge this?

**Comment by agitter** [comment]
It looks like we can't enable GitHub pages on this repository until after the merge.  The option is disabled right now:
image

File: README.md (hunk 1 above)
Comment (thevaachandereng) [reply README.md]: sure. I will make another pull request once the site goes live.

**Comment by thevaachandereng** [comment]
Yes. I tried it too. I suggest we merge and make another pull request.

**Pull request merged**